pkg/xray/policies.go

```diff
@@ -67,7 +67,7 @@ type Policy struct {
 	Modified    *string       `json:"modified,omitempty"`
```

Comment:
Per our discussion last week, please remove the use of pointers and `omitempty`. @chb0github

Reply:
It's one of those things that just keeps on giving

Reply:
removed